### PR TITLE
feat: Added insecure registry support

### PR DESCRIPTION
**Pull request by podnov**
Added --insecure-registry=[true|false] as a command line flag
when adding a docker-registry account.

This is dependent on spinnaker/clouddriver#1887 and will
require a dependency version bump once merged and released.

This fixes spinnaker/halyard#680.

### CODE REVIEW

File: halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/dockerRegistry/DockerRegistryEditAccountCommand.java [hunk 1]
```diff
@@ -88,6 +88,12 @@ protected String getProviderName() {
   )
   private Long cacheIntervalSeconds;
 
+  @Parameter(
+      names = "--insecure-registry",
+      description = DockerRegistryCommandProperties.INSECURE_REGISTRY_DESCRIPTION
```
Comment (lwander) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/dockerRegistry/DockerRegistryEditAccountCommand.java]: Can you make the arity=1 here? This allows you to set the flag back to false.

Comment (podnov) [reply halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/dockerRegistry/DockerRegistryEditAccountCommand.java]: I've set arity=1 for both the add and edit command.  Were you also suggesting setting the default value of the flag to false?  I intentionally left that default null in the edit command for the isSet() check.

**Comment by podnov** [comment]
Figured how to point this at the a new version of cloudriver for the api changes. This should be gtg as far as I'm concerned.

**Comment by podnov** [comment]
At kubecon watching the keynote about spinnaker, hoping to get this merged soon.

**Comment by lwander** [comment]
Sorry! nice work

**Pull request merged**